--- knotes-17.11.90/src/configdialog/knotecollectionconfigwidget.cpp.omv~	2017-12-02 21:30:09.588503064 +0100
+++ knotes-17.11.90/src/configdialog/knotecollectionconfigwidget.cpp	2017-12-02 21:30:18.433522055 +0100
@@ -148,7 +148,7 @@ KNoteCollectionConfigWidget::KNoteCollec
     mCollectionFilter = new KRecursiveFilterProxyModel(this);
 #else
     mCollectionFilter = new QSortFilterProxyModel(this);
-    mCollectionFilter->setRecursiveFiltering(true);
+    mCollectionFilter->setRecursiveFilteringEnabled(true);
 #endif
     mCollectionFilter->setSourceModel(mDisplayNotifierProxyModel);
     mCollectionFilter->setDynamicSortFilter(true);
